config/autoload/zend-expressive.global.php

```diff
@@ -3,9 +3,6 @@
 return [
     'debug' => false,
 
-    // Cache the configuration. Recommended for production.
-    'config_cache_enabled' => false,
```

Comment:
Since we'll be putting the application in development mode on initial install, I'm wondering if we should have this enabled by default? Particularly if the development configuration disables it...